Edits Governance - MERGE ON THE 15th OF JUNE (#4823)
w3f/polkadot-wiki: docs/learn/learn-opengov.md

in its lifecycle. As updates for OpenGov are making their way to the networks, plans for future
updates are already in development.

For technical information about OpenGov and how to interact with it, please refer to this
[dedicated Wiki page](../maintain/maintain-guides-opengov.md).

:::

> :::info Delegate your votes and make your voice count!

Reviewer: ```suggestion
:::info Delegate your votes
```

In OpenGov you can delegate your votes to different entities, who will vote on your behalf. You can

> easily delegate your votes using the

Reviewer: ```suggestion
delegate your votes using the
```

[**Polkadot Delegation Dashboard**](https://delegation.polkadot.network/). See

> [this video tutorial](https://www.youtube.com/watch?v=tlh_KmWQiSQ) to learn everything about the

Reviewer: ```suggestion
[this video tutorial](https://www.youtube.com/watch?v=tlh_KmWQiSQ) to learn about the features of the
```

dashboard.

> If you are already staking, you can use bonded tokens for voting. Note that if you are voting with
> conviction (not possible with the Delegation Dashboard), your tokens will have a democracy lock in
> addition to the staking lock. For more information about locks, see
> [this page](./learn-accounts.md/#unlocking-locks).

Reviewer: ```suggestion
If you are staking directly and not through a nomination pool, you can use bonded tokens for voting. Note that if you are voting with
conviction, your tokens will have a democracy lock in
addition to the staking lock. For more information about locks, see
[this page](./learn-accounts.md/#unlocking-locks).
```

- Dissolving the current [Council](./learn-governance.md#council) collective
- Allowing users to delegate voting power in more ways to community members
- Dissolving the [Technical Committee](./learn-governance.md#technical-committee) and establishing
  the broader [Polkadot Technical Fellowship](#the-technical-fellowship)

The figure below shows an overview of OpenGov's structure.

:::info

See [this page](./learn-governance.md#summary) for a comparison with the structure of Governance V1.

:::

![opengov-overview](../assets/opengov-overview.png)

> In OpenGov, a proposal can only be submitted by the Public. The proposal will enter a Lead-in period

Reviewer: ```suggestion
In OpenGov, all the proposals are initiated by the public. The proposal will enter a Lead-in period
```

(for more information, see [Referenda Timeline](#referenda-timeline)), after which it will follow a

> specific [Track](#origins-and-tracks) depending on its Origin. There are

Reviewer: ```suggestion
specific [Track](#origins-and-tracks) which has a dedicated Origin. There are
```

[15 Origins](../maintain/maintain-guides-opengov.md#origins-and-tracks-info), each with a different

> track. Different origins shape the duration of a referendum as well as how many referenda can be

Reviewer: ```suggestion
track. The origins and tracks parameters are preset values that set the duration of a referendum as well as how many referenda can be
```

> voted on simultaneously. For example, a treasury proposal can now have different origins depending
> on the amount to be spent and the type of proposal. A small tip will use the Small Tipper origin,

Reviewer: ```suggestion
voted on simultaneously. For example, a treasury proposal can now be submitted in different tackss depending
on the amount requested. A proposal for a small tip will need to be submitted in the Small Tipper track,
```

> while funding for a conference requiring substantial funds will use the Medium or Big Spender

Reviewer: ```suggestion
while a proposal requiring substantial funds will need to be submitted to the Medium or Big Spender
```

> origin.

Reviewer: ```suggestion
track.
```

The [Polkadot Technical Fellowship](#the-technical-fellowship) can decide to

> [whitelist](#whitelisting) a proposal that will end in the Whitelist Caller origin. Those proposals

Reviewer: ```suggestion
[whitelist](#whitelisting) a proposal that will be enacted through the Whitelist Caller origin. Those proposals
```

> will have a shorter Lead-in, Confirmation, and Enactment period.

Reviewer: ```suggestion
will have a shorter Lead-in, Confirmation, and Enactment period when compared to the Root Origin track.
```

Voting will follow an [Approval and Support](#approval-and-support) curve that will differ depending
on the origin. There is a confirmation period during which approval and support criteria must be

> held within the voting period. If the proposal passes, it will go through an enactment period.

Reviewer: ```suggestion
Each track has its own preset [Approval and Support](#approval-and-support) curves which are based
on the origin's privileges. When both the approval and support criteria are
satisfied for a specific period (called the confirmation period), the referenda passes and will be executed after the enactment period.
```

> All proposals within each origin and across origins can be voted on simultaneously (assuming track

Reviewer: ```suggestion
All referenda within each track and across tracks can be voted on simultaneously (assuming track
```

maximum capacity is not reached).

OpenGov comes also with multi-role delegations where token holder can assign different tracks to
different entities they trust and that might be expert in judging proposals within specific origins
only. In this way token holders do not need to be up-to-date with governance matters, and can still

> make their vote count through delegates.

Reviewer: ```suggestion
OpenGov also comes with multi-role delegations where the token holder can assign voting power on different tracks to
different entities who are experts in judging the referenda submitted to those tracks. For example, suppose a token holder does not have the technical background to consider the merits and vote on the referenda submitted to the Root track. In that case, they can delegate their voting power just for the Root track to a trusted expert who (according to them) acts in the best interest of the network protocol. In this way, token holders do not need to be up-to-date with governance matters and can still
make their votes count through delegates.
```

[Delegating Voting Power](./../maintain/maintain-guides-opengov.md#delegating-voting-power) section
on the [OpenGov Maintenance](./../maintain/maintain-guides-opengov.md) page.

## The Technical Fellowship

:::info From Technical Committee to the Technical Fellowship

> The Polkadot Technical Fellowship is a collection of Substrate experts. The Fellowship was

Reviewer 2: ```suggestion
The Polkadot Technical Fellowship is a collection of Substrate experts.  This fellowship was
```

> established in 2022. In OpenGov, the Fellowship replaces the

Reviewer 2: ```suggestion
established in 2022. In OpenGov, this fellowship replaces the
```
